pkg/admission/imagepolicy/imagepolicy.go

```diff
@@ -198,7 +199,16 @@ func (a *ImagePolicyPlugin) admit(ctx context.Context, attr admission.Attributes
 	if obj, ok := attr.GetObject().(metav1.Object); ok {
 		for _, ownerRef := range obj.GetOwnerReferences() {
 			if ownerRef.Controller != nil && *ownerRef.Controller {
-				klog.V(5).Infof("skipping image policy admission for %s:%s/%s, reason: has controller owner reference", attr.GetKind(), attr.GetNamespace(), attr.GetName())
+				gv, err := schema.ParseGroupVersion(ownerRef.APIVersion)
+				if err != nil {
+					return err
+				}
+				gk := schema.GroupKind{Group: gv.Group, Kind: ownerRef.Kind}
+				if !imagereferencemutators.GetPodTemplateSupportedGroupKinds().Has(gk) {
```

Comment:
one is disadvantage is that this could break 3rd party controllers that do not expect the pods to be changed on admission

Reply:
we could document this. In any case I would assume that those managing the external controllers would need to be aware of this.

Reply:
> one is disadvantage is that this could break 3rd party controllers that do not expect their owned pods to be changed on admission

does this make this a breaking change? 🤔